02-lsp_extension/02.00-lsp_intro.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
<p>Really nice metaphor! Did you you come up with that all by yourself?</p><p> </p><p>As for **Finding References** I'm not really clear on how this is differnent from "go to definition". Like what is a symbol or identifier?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nicole-brewer/jupyterlab-ide/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>